src/LibHdb++MySQL.cpp

```diff
@@ -2361,1496 +2323,366 @@ template <typename Type> void HdbPPMySQL::store_array(string attr, vector<Type>
 
 //=============================================================================
 //=============================================================================
-template <typename Type> void HdbPPMySQL::store_array_json(string attr, vector<Type> value_r, vector<Type> value_w, int quality/*ATTR_VALID, ATTR_INVALID, ..*/, string error_desc, int data_type/*DEV_DOUBLE, DEV_STRING, ..*/, int write_type/*READ, READ_WRITE, ..*/, Tango::AttributeDimension attr_r_dim, Tango::AttributeDimension attr_w_dim, double ev_time, double rcv_time, string table_name, enum_field_types mysql_value_type, bool isNull)
+template <typename Type> void HdbPPMySQL::store_array_json(const vector<event_values_param<Type> > &event_values, int data_type/*DEV_DOUBLE, DEV_STRING, ..*/, int write_type/*READ, READ_WRITE, ..*/, const string & table_name, enum_field_types mysql_value_type, bool is_unsigned)
 {
+	uint32_t max_size = event_values.size();
+	uint32_t insert_size = (max_size <= batch_size) ? max_size : batch_size;
```

Comment:
I am still a bit sceptical on the use of uint32_t. And what about using std::min ?